### ISSUE TITLE
Documentation should specify a version requirement for `hide_env_diff`

### ISSUE THREAD

**Issue by RandomSpaceship** [direnv/direnv#1370]
The [man page](https://direnv.net/man/direnv.toml.1.html) references the `hide_env_diff` option, but does not mention that it requires a recent (v2.34.0 or later, from March 2024) version of `direnv` (in comparison to the `load_dotenv` option which does). This can be confusing for people using distros with longer release cycles (I initially encountered this issue with Ubuntu 22.04, which ships 2.25.2), as the option does not work. I've also updated the [wiki](https://github.com/direnv/direnv/wiki/Quiet-or-Silence-direnv) with the option.

**Comment by tambry** [comment]
Wouldn't just using the manpage shipped with your installed version obviate this issue?

**Comment by RandomSpaceship** [comment]
No. I specifically ran into this issue because I'm managing a project which needs to run on multiple distros, each of which are shipped with different versions.
